### PR TITLE
Added open-cobol to linux builds

### PR DESCRIPTION
**Pull request by LuisEnMarroquin**
# Description
New tool,
Added open-cobol to your linux hosts, both Ubuntu 16.04 and 18.04
  
**For new tools, please provide total size and installation time.**
~5 seconds, is very lightweight

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/Ubuntu1604-README.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ Local version: Unknown)
 - rustfmt (1.4.11-stable)
 - clippy (0.0.212)
 - rustdoc (1.42.0)
+- open-cobol (1.1.0)
```
Comment (al-cheb) [inline images/linux/Ubuntu1604-README.md:203]: Could you please  remove changes in autoupdated documentation?

File: images/linux/Ubuntu1804-README.md [hunk 2]
```diff
@@ -196,6 +196,7 @@ Local version: Unknown)
 - rustfmt (1.4.11-stable)
 - clippy (0.0.212)
 - rustdoc (1.42.0)
+- open-cobol (1.1.0)
```
Comment (al-cheb) [inline images/linux/Ubuntu1804-README.md:199]: Could you please  remove changes in autoupdated documentation?

File: images/linux/scripts/installers/open-cobol.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+################################################################################
+##  File:  open-cobol.sh
+##  Desc:  Installs open-cobol
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install open-cobol
+apt-get update
```
Comment (al-cheb) [inline images/linux/scripts/installers/open-cobol.sh:11]: We don't need to update once more time here.

File: images/linux/scripts/installers/open-cobol.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+apt-get install gcc open-cobol -y
```
Comment (al-cheb) [inline images/linux/scripts/installers/open-cobol.sh:12]: Please remove gcc.

File: images/linux/scripts/installers/open-cobol.sh [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+# Run tests to determine that the software installed as expected
+echo "Testing open-cobol installation"
+if ! command -v cobc; then
+    echo "open-cobol was not installed"
+    exit 1
+fi
+
+# Document what was added to the image
+echo "Lastly, document the installed version"
+DocumentInstalledItem "Open-Cobol ($(echo $(cobc -V) | cut -c 18-22))"
```
Comment (al-cheb) [inline images/linux/scripts/installers/open-cobol.sh:24]: I don't think we should changed official name OpenCOBOL to Open-Cobol. Could you please update to ?
`DocumentInstalledItem "OpenCOBOL ($(echo $(cobc -V) | cut -d ' ' -f3))"`

**Comment by maxim-lobanov** [comment]
@LuisEnMarroquin, thank you for contribution!
Your changes look good and checks passed successfully. But we can't merge the pull-request right now.
We have an active incident with disk space on Ubuntu images so we have to wait a bit until it is resolved.

**Comment by alepauly** [comment]
Hi @LuisEnMarroquin! As @maxim-lobanov mentioned we've been having disk space issues in the ubuntu environments so at this time we rather not add tools that are not essential for a large number of workflows. Thanks a lot for your contribution. I'll close the PR but we can reopen in the future if the situation changes.

**Pull request closed without merging**